anti_kd_backdoor/data/dataset/cifar.py

```diff
@@ -117,3 +118,69 @@ def __init__(self, *, ratio: float, poison_label: int, **kwargs) -> None:
         RatioPoisonLabelDataset.__init__(self,
                                          ratio=ratio,
                                          poison_label=poison_label)
+                            
+
+    
+class GTSRB(TORCH_GTSRB, DatasetInterface):
+data: np.ndarray
+targets: list[int]
+
+def get_xy(self) -> XY_TYPE:
+    return list(self.data), self.targets.copy()
+
+def set_xy(self, xy: XY_TYPE):
+    x, y = xy
+    assert len(x) == len(y)
+
+    self.data = np.stack(x, axis=0)
+    self.targets = y.copy()
+
+@property
+def num_classes(self) -> int:
+    return len(set(self.targets))
+
+@property
+def raw_num_classes(self) -> int:
```

Comment:
`raw_num_classes` of gtsrb should be 43
![image](https://user-images.githubusercontent.com/44188071/199002066-a1791894-1488-4c2a-9037-22f8b28cd942.png)